Tudat/Astrodynamics/Propagators/UnitTests/unitTestCustomStatePropagation.cpp

```diff
@@ -0,0 +1,221 @@
+/*    Copyright (c) 2010-2016, Delft University of Technology
+ *    All rigths reserved
+ *
+ *    This file is part of the Tudat. Redistribution and use in source and
+ *    binary forms, with or without modification, are permitted exclusively
+ *    under the terms of the Modified BSD license. You should have received
+ *    a copy of the license with this file. If not, please or visit:
+ *    http://tudat.tudelft.nl/LICENSE.
+ */
+
+#define BOOST_TEST_MAIN
+
+#include <boost/bind.hpp>
+#include <boost/make_shared.hpp>
+#include <boost/shared_ptr.hpp>
+#include <boost/test/unit_test.hpp>
+
+#include <Eigen/Core>
+
+#include <Tudat/Basics/testMacros.h>
+#include <Tudat/Astrodynamics/BasicAstrodynamics/orbitalElementConversions.h>
+#include <Tudat/Astrodynamics/BasicAstrodynamics/physicalConstants.h>
+#include <Tudat/Astrodynamics/BasicAstrodynamics/unitConversions.h>
+#include <Tudat/Mathematics/NumericalIntegrators/rungeKutta4Integrator.h>
+#include <Tudat/Astrodynamics/BasicAstrodynamics/stateVectorIndices.h>
+#include <Tudat/Mathematics/BasicMathematics/linearAlgebraTypes.h>
+#include <Tudat/InputOutput/basicInputOutput.h>
+
+#include "Tudat/SimulationSetup/PropagationSetup/dynamicsSimulator.h"
+#include <Tudat/External/SpiceInterface/spiceInterface.h>
+#include <Tudat/SimulationSetup/EnvironmentSetup/body.h>
+#include <Tudat/SimulationSetup/EnvironmentSetup/createBodies.h>
+#include "Tudat/SimulationSetup/PropagationSetup/createNumericalSimulator.h"
+
+namespace tudat
+{
+
+namespace unit_tests
+{
+
+using namespace tudat::propagators;
+using namespace tudat::simulation_setup;
+using namespace tudat::numerical_integrators;
+
+BOOST_AUTO_TEST_SUITE( test_body_mass_propagation )
+
+double getDummyCustomState1(
+        const double currentCustomState, const double currentTime )
```

Comment:
Arguments should be flipped. First argument is the currentTime and second argument is the currentCustomState.